cmd/commands/team.go

```diff
@@ -52,6 +52,14 @@ Permanently deletes a team along with all related information including posts fr
 	RunE:    deleteTeamsCmdF,
 }
 
+var ListTeamsCmd = &cobra.Command{
+	Use:     "list",
+	Short:   "List all teams.",
+	Long:    `List all teams.`,
```

Comment:
Propose long description: `List all teams on the server.`

Reply:
Done :)